tests/utils/check.rs

```diff
@@ -62,11 +62,21 @@ pub fn info(escrow_program: &Program, wallet_id: u128, wallet_info: Wallet) {
     )
 }
 
-pub fn created_wallets(escrow_program: &Program, created_wallets: Vec<(WalletId, Wallet)>) {
-    assert_eq!(
-        escrow_program
-            .meta_state::<_, EscrowStateReply>(EscrowState::CreatedWallets)
-            .unwrap(),
-        EscrowStateReply::CreatedWallets(created_wallets)
-    );
```

Comment:
You can leave it unchanged, and just swap wallets here:
https://github.com/gear-dapps/escrow/blob/9b627bbf771057fa5642ba89ea1da5660532c383/tests/other.rs#L206-L223

Reply:
We don't know true order in hashmap.

I think this is correct. We need sorting to order because in the future if we extend the test, we will again have to place the elements in the correct order.